### PR TITLE
Creating inline validator

### PR DESCRIPTION
**Pull request by aaronbhansen**
This allows you to create a validator inline that validates with a custom function. 

This will help when you need to create a one off validation without having to create a custom validator.

### CODE REVIEW

File: tests/unit/validators/date-test.js [hunk 1]
```diff
@@ -217,7 +217,8 @@ test('after now or on', function(assert) {
   assert.expect(3);
   var now = moment().format('MMM Do, YYYY');
   options = {
-    onOrAfter: 'now'
+    onOrAfter: 'now',
+    percision: 'second'
```
Comment (aaronbhansen) [inline tests/unit/validators/date-test.js]: I noticed the milliseconds would randomly fail, when both values were now. This fixed it.

**Comment by offirgolan** [comment]
This is already supported via validator(function(...))

**Comment by aaronbhansen** [comment]
Oh nice, not sure how I missed that. 

Let me know if you want a pull request for the date test fix, or if you want to just add it in.

**Comment by offirgolan** [comment]
If you can create a quick pr for it, I can merge it. Why was it failing for milliseconds and not seconds?

**Comment by offirgolan** [comment]
And no worries 😸 Thanks for taking the time to put this together!

**Pull request closed without merging**